platform/windows/common.cmake

```diff
@@ -26,22 +27,22 @@ set(file_name_list
 
 foreach(directory_name ${directory_name_list})
   install(
-    DIRECTORY "${OSQUERY_DATA_PATH}/Program Files (x86)/osquery/${directory_name}"
+    DIRECTORY "${OSQUERY_DATA_PATH}/Program Files/osquery/${directory_name}"
```

Comment:
@alessandrogario or @directionless do you know if us migrating to the non-`(x86)` Program Files is going to cause any issues? I feel like we should default to `Program Files` as we typically generate the x64 based binaries yes?

Reply:
The internet says:
> Starting with Windows Vista, the 64-bit versions of Windows have two Program Files folders. The regular Program Files folder holds 64-bit applications, while "Program Files (x86)" is used for 32-bit applications.

So I think I think it is correct to drop the `(x86)`

Reply:
:) Never did trust that "internet"